CMakeLists.txt

```diff
@@ -118,13 +118,7 @@ else ()
 endif ()
 
 if (OE_SGX)
-  if (WIN32)
-    # Building enclaves on windows is on by default but can be disabled for enclaves pre-compiled under linux
-    option(BUILD_ENCLAVES "Build ELF enclaves" ON)
-  else ()
-    set(BUILD_ENCLAVES ON)
-  endif ()
-
+  set(BUILD_ENCLAVES ON)
```

Comment:
On Windows, we still want that `BUILD_ENCLAVES` can be overwritten. Can you revert the changes?